process_group/process_group_windows.go

```diff
@@ -41,6 +41,10 @@ func (pg *winProcessGroup) RootProcess() *exec.Cmd {
 	return pg.root
 }
 
+func (pg *winProcessGroup) Run() *exec.Cmd {
```

Comment:
The type in `process_group_unix.go` and in `process_group_windows.go` do not match,

```
func (pg *unixProcessGroup) Run() error {
```

vs

```
func (pg *winProcessGroup) Run() *exec.Cmd {
```

Reply:
done